docs/streams/developer-guide.html

```diff
@@ -1589,7 +1592,7 @@
                             return aggValue - oldValue;
                           }
                         },
-                        "reduced-table-store" /* state store name */);
+                    );
```

Comment:
You must remove the comma `,` in the line before, now that we removed the last parameter.